modules/components/pages/inputs/redpanda_migrator.adoc

```diff
@@ -705,6 +708,32 @@ The label of the `redpanda_migrator` output in which the currently selected topi
 
 *Type*: `string`
 
-*Default*: `"redpanda_migrator_output"`
+*Default*: `redpanda_migrator_output`
+
+=== `replication_factor_override`
+
+Use the value specified in the `replication_factor` field when creating topics.
```

Comment:
```suggestion
Indicate whether to override the replication factor of the input topics when creating copies of them. The new replication factor must be specified in the <<replication-factor, `replication_factor` field>>.
```

@Jeffail or @mihaitodor I'm unsure why the input has this option. Does the input or the output create the topic copies?

Reply:
Also, I assume if this is `false` that the new topics inherit the replication factor of the original. Is that right?

Reply:
Updated as requested. And yes that's right.

Reply:
Yep, that makes sense. I might add “ when creating copies of them in the destination”.

Reply:
LGTM, thank you!